pallets/cash/src/core.rs

```diff
@@ -734,7 +745,11 @@ pub fn liquidate_internal<T: Config>(
         )?;
 
     require!(
-        has_liquidity_to_reduce_asset_with_added_collateral::<T>(
+        !liquidity::has_non_negative_liquidity::<T>(borrower)?,
+        Reason::SufficientLiquidity
```

Comment:
😹

Reply:
I might change it, but it just feels so right.